linfa-clustering/src/dbscan/hyperparameters.rs

```diff
@@ -0,0 +1,60 @@
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
+/// The set of hyperparameters that can be specified for the execution of
+/// the [DBSCAN algorithm](struct.Dbscan.html).
+pub struct DbscanHyperParams {
+    /// Distance between points for them to be considered neighbours.
+    tolerance: f64,
```

Comment:
`DbscanHyperParams`'s fields are private - we need to copy these doc comments over to corresponding methods, both in the builder and accessor methods.
We should also document the default value in the builder methods.

Reply:
Done :smile: